app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -71,9 +71,23 @@ def drop_api_namespace(original_request_path)
     end
 
     def get
-      r = Resources::Candlepin::Proxy.get(@request_path)
-      logger.debug filter_sensitive_data(r)
-      render :json => r
+      extra_headers = {}
+      modified_since = request.headers['If-Modified-Since']
```

Comment:
Might be good to move 'if-modified-since' to a constant. Maybe RestClient already has one to use (although I have no issues creating our own in this file).